bot/seasons/evergreen/bookmark.py

```diff
@@ -46,6 +63,25 @@ def __init__(self, bot: commands.Bot):
         else:
             await ctx.message.add_reaction(Emojis.envelope)
 
+        embed.add_field(
+            name=f'Bookmarked from {ctx.author.name}.',
+            value=f'[Visit original message]({ctx.message.jump_url})',
+            inline=False
+        )
+
+        def check(reaction: str, user: str) -> None:
+            return str(reaction.emoji) == Emojis.envelope
+
+        sent_person = [ctx.author.id]  # list of id who got the message
+        while True:
+            try:
+                reaction, user = await self.bot.wait_for("reaction_add", timeout=360.0, check=check)
+                if (str(user.name) != str(self.bot.user.name)) and (user.id not in sent_person):

```

Comment:
Why the mix between string and ID comparisons? Per above, use direct comparisons between the `Member` objects.